generate.py

```diff
@@ -1,11 +1,12 @@
-import sys
+import sys, shutil
+import typing
```

Comment:
Not sure if this won't mess up the homebrew installation, because `typing` might be something that would need to be imported in the formula.. (I also don't necessarily like how it looks 🙈)